core/api/serializers/country_programme.py

```diff
@@ -35,25 +40,92 @@ class Meta:
 
 
 class CPRecordSerializer(serializers.ModelSerializer):
-    blend = serializers.StringRelatedField()
-    substance = serializers.StringRelatedField()
+    chemical_name = serializers.SerializerMethodField()
+    chemical_group_name = serializers.SerializerMethodField()
     record_usages = CPUsageSerializer(many=True)
```

Comment:
The usage name is not enough in CPUsageSerializer. There can be multiple usages with name "Other" (check China/2021/HFC-32)